Jenkinsfile

```diff
@@ -83,12 +83,19 @@ pipeline {
     }
   }
   post {
-        always {
-            archive "coverage/**"
-            publishHTML(target: [reportDir: 'coverage', reportFiles: 'index.html', reportName: 'Coverage', keepAll: true])
-            junit 'log/*.xml'
-            sh 'make clean'
-            deleteDir() /* clean up our workspace */
+    always {
+      archive "coverage/**"
+      publishHTML(target: [reportDir: 'coverage', reportFiles: 'index.html', reportName: 'Coverage', keepAll: true])
+      junit 'log/*.xml'
+      sh 'make clean'
+      deleteDir() /* clean up our workspace */
+    }
+    failure {
+      script {
+        if (env.BRANCH_NAME == 'master') {
+          slackSend message: "Failed vets-api CI on branch: [${env.BRANCH_NAME}]", color: 'danger', failOnError: true
```

Comment:
Can you add a `${env.RUN_DISPLAY_URL}".stripMargin()` to the string here? That'll give us a link to the output page when it does fail to minimize the clicking around.